lib/res_util/tests/es_testdata.cpp

```diff
@@ -49,6 +70,71 @@ res::es_testdata make_testdata(int ens_size, int obs_size) {
   return td;
 }
 
+void test_deactivate() {
+  int ens_size = 10;
+  int obs_size =  7;
+  int del_iobs = 3;
+  int del_iens = 7;
+  res::es_testdata td1 = make_testdata(ens_size, obs_size);
+  res::es_testdata td2 = make_testdata(ens_size, obs_size);
+
+
+
+  test_assert_throw(td1.deactivate_obs(10), std::invalid_argument);
+  td1.deactivate_obs(del_iobs);
+
+  for (int i1=0; i1 < matrix_get_rows(td1.R); i1++) {
+    int i2 = i1;
```

Comment:
Would prefer this and the similar construction below to read:
`int i2 = i1 + (i1 >= del_iobs ? 1 : 0);`

(or if you insist `int i2 = i1 + (i1 >= del_jobs);`, but I personally prefer the first one).

Reply:
Ternary operator - that is beyond my pay grade; but I'll Copy&Paste your suggestion.